parsons/google/google_sheets.py

```diff
@@ -293,6 +293,55 @@ def append_to_sheet(
         sheet.update_cells(cells, value_input_option=value_input_option)
         logger.info(f"Appended {table.num_rows} rows to worksheet.")
 
+    def paste_data_in_sheet(
+        self, spreadsheet_id, table, worksheet=0, header=True, startrow=0, startcol=0
+    ):
+        """
+        Pastes data from a Parsons table to a Google sheet. Note that this may overwrite
```

Comment:
Might add a little here to make sure the user knows that what's special about this is it can paste to anywhere in the sheet (can even reference overwrite and append to show when you'd use this instead of those)